src/contracts/libraries/SafetyBits.sol

```diff
@@ -101,6 +101,9 @@ library SafetyBits {
     {
         self.lockState = _LOCK_PAYMENTS;
         self.addressPointer = addressPointer;
+        unchecked {
+            ++self.callIndex;
+        }
```

Comment:
@BenSparksCode this is the recommended change by the auditors. Should we remove the `unchecked`?

Reply:
I think let's keep it. It saves a little bit of gas if we use `unchecked`.

But we should be aware that the `callIndex` var is only `uint8` which means it has a max value of 255 (2^8 - 1) before overflowing in the unchecked increment. To prevent that, maybe we should adjust `_MAX_SOLVERS` in AtlasVerification.

Currently `_MAX_SOLVERS` is `type(uint8).max - 2`, so if we change it to `type(uint8).max - 4` then we can be sure that we will never overflow in the unchecked block because it would have been caught in the `_validCalls()` check in AtlasVerification.

That `4` magic number which represents the number of calls excluding solver calls should also be a constant somewhere so we only change the value in one place.

So that's max 251 solvers which seems more than enough.

Will make the changes now.